### PR TITLE
Disabling user can edit survey when push returns response code 403.

### LINKED ISSUE #1720: On 403 response consider config file obsolete
When a voucher is sent to the WS, a new answer code will be introduced (403) to indicate the config file version used to push that voucher is obsolete and the app needs to be upgraded. 

In this situation, just like when the app's not upgraded to the last version, the + button to create a new survey disappears, and the user tries to upgrade the app to the last version of the config files.

### PR DESCRIPTION
**Pull request by manuelplazaspalacio**
Close #1720

### CODE REVIEW

**Comment by manuelplazaspalacio** [comment]
Close #1720

File: app/src/androidTestEreferrals/java/org/eyeseetea/malariacare/SettingsActivityShould.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+import org.eyeseetea.malariacare.data.database.utils.PreferencesState;
+import org.eyeseetea.malariacare.services.PushService;
+import org.eyeseetea.malariacare.services.strategies.PushServiceStrategy;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+public class SettingsActivityShould {
```
Comment (ifoche) [inline app/src/androidTestEreferrals/java/org/eyeseetea/malariacare/SettingsActivityShould.java:22]: This test is failing for me. Can you take a look @manuelplazaspalacio ?

**Review by ifoche — CHANGES_REQUESTED** [review]
@manuelplazaspalacio there's a failing tests, can you check it please? Thanks!

**Review by ifoche — APPROVED** [review]
thanks @manuelplazaspalacio, I've created the separate issue #1767 to deal with the SettingsActivityShould problem but accepting this code

**Pull request merged**